server_plugin/server.py

```diff
@@ -156,6 +157,11 @@ def start(vca_client, **kwargs):
             raise cfy_exc.NonRecoverableError("Could not create vApp")
         wait_for_task(vca_client, task)
 
+    if not _get_state(vca_client, ctx):
+        return ctx.operation.retry(
+            message='Waiting for status',
```

Comment:
Message could be more descriptive, something like "Waiting for VM's configuration to complete"